ObjectOrientedProgramming/Memory Ownership/new and delete operators/task.md

```diff
@@ -0,0 +1,43 @@
+New and delete operators are used for dynamic memory allocation in C++. The new operator allocates memory on the heap for an object or an array of objects. The delete operator releases the memory back to the heap.
```

Comment:
It is important to highlight the difference w.r.t. `malloc` and `free`: 
i.e. the fact that `new` and `delete` also call constructor/destructor of an object.